resources/resource.py

```diff
@@ -0,0 +1,81 @@
+from flask_restful import Resource, reqparse
+from bson import json_util
+from bson.objectid import ObjectId
+from flask_jwt_extended import (
+    jwt_refresh_token_required,
+    get_jwt_identity,
+    jwt_required,
+    get_raw_jwt,
+)
+
+from db import mongo
+
+_resource_parser = reqparse.RequestParser()
+_resource_parser.add_argument(
+    "resource_name", type=str, required=True, help="This field cannot be blank."
+)
+_resource_parser.add_argument(
+    "standard", type=list, required=True, help="This field cannot be blank."
+)
+
+
+class ResourceRegister(Resource):
+    def post(self):
+        data = _resource_parser.parse_args()
+
+        try:
+            resource = mongo.db.resource.find_one(
+                {"resource_name": data["resource_name"]})
+        except:
+            return {"message": "An error occurred looking up the resource_name"}, 500
```

Comment:
you are looking up the resource and not a resources_name(you are just querying with resource_name which is actually supposed to be called name)